core/api.py

```diff
@@ -119,3 +137,76 @@ def by_date(self, date:str = 'today', period: str = '1d' ) -> dict:
         url += f"/activities/heart/date/{date}/{period}.json"
         return api_request(url, self._tokens)
 
+    def by_interval_intraday(
+        self,
+        start_date: str = 'today',
+        end_date: str = 'today',
+        detail_level: str = '1min',
+        start_time: str = None,
+        end_time: str = None,) -> dict:
+        """Heart rate intraday by interval.
+
+        Time parameters are optional.
+
+        API endpoint returns inconsistent results depending on the parameters.
+        The following table documents the results of different parameter combinations:
+
+        start_date   | end_date     | detail_level | start_time | end_time | result
+        ---------------------------------------------------------------------
+        'today'      | 'today'      | '1min'       | None       | None     | daily + intraday

```

Comment:
For the result column where it says intraday (but not intraday at 1 sec intervals), should it say intraday at 1 min intervals so it's consistent?

Reply:
done